dist/_patterns/01-components/01-page-elements/hero.twig

```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+  <div class="hero__content" role=”banner”>
+    <h1 class="hero__header">{{ pageTitle }}</h1>
+    <h2 class="hero__subheader">{{ pageSubTitle }}</h2>
```

Comment:
I’m leaning towards using a {{ subtitle }} variable instead of {{ pageSubTitle }}.